src/integration/resources/docker/coordinator.go

```diff
@@ -21,26 +21,16 @@
 package docker
 
 import (
-	"bytes"
```

Comment:
Again, most of this has been moved to `src/integration/resources/common/coordinator_client.go`